### PR TITLE
Kf gene expression in tumors

### PR DESCRIPTION
**Pull request by wnkhan**
Added node for gene expression in tumors.

Input: GeneInfo
Output: GeneExpressionInTumor

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203977799246430
  - https://app.asana.com/0/0/1204374829353164
  - https://app.asana.com/0/0/1204432748886278

### CODE REVIEW

File: components/service/kf/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "kf",
+  "version": "1.0.0",
+  "license": "unk",
```
Comment (wnkhan) [inline components/service/kf/package.json]: Get feedback from the team.

File: components/core/input/tumor/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "tumor",
+  "version": "1.0.0",
+  "license": "CC-BY-NC-SA-4.0",
```
Comment (wnkhan) [inline components/core/input/tumor/package.json:4]: Get feedback from the team.

File: components/core/input/primitives.ts [hunk 3]
```diff
@@ -132,3 +132,17 @@ export const Tissue = {
...
+
+export const TumorGeneExpression = {
+  name: 'TumorGeneExpression ',
+  label: 'TumorGeneExpression ',
+  icon: [tissue_icon],
+  color: '#98D7C2',
+  extra: {
+    term: {
+      meta: {
+        example: 'Brain',
```
Comment (u8sand) [inline components/core/input/primitives.ts]: This should probably be a tumor name as an example, perhaps `Meningioma`, for instance.

File: components/service/kf/__init__.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+    api_url = 'https://openpedcan-api.d3b.io/tpm/gene-all-cancer/json'
+    query_params = {'ensemblId':ensembl_id,'includeTumorDesc':'primaryOnly'}
+
+    with requests.Session() as session:
+        response = session.get(api_url,params=query_params)
+        response.raise_for_status()
+        
+
+    request_df = pd.read_json(response.text,'records')
```
Comment (u8sand) [inline components/service/kf/__init__.py]: Please address deprecation warning
```
components/service/kf/__init__.py:25: FutureWarning: Starting with pandas version 2.0 all arguments of read_json except for the argument 'path_or_buf' will be keyword-only.
  request_df = pd.read_json(response.text,'records')
```

Comment (wnkhan) [reply components/service/kf/__init__.py]: Addressed warning with https://github.com/nih-cfde/playbook-partnership/pull/38/commits/867069d0a744fc418e9b6a26c5c36dccb9eccaca.

File: components/core/input/primitives.ts (hunk 3 above)
Comment (wnkhan) [reply components/core/input/primitives.ts]: Corrected copy pasta error with https://github.com/nih-cfde/playbook-partnership/pull/38/commits/06be34924211e23c6a4429ffb00282e73bcf78ae.

**Comment by wnkhan** [comment]
Branch is not ready to be merged. After synchronizing my fork and merging main with my branch, the build broke. I'll need to resolve this issue before able to merge. Only mentioning this here because I marked it 'Ready for Review' before I realized the issue with the build.

**Pull request merged**